cmd/skopeo/utils.go

```diff
@@ -196,6 +198,7 @@ func (opts *imageDestOptions) newSystemContext() (*types.SystemContext, error) {
 	}
 
 	ctx.DirForceCompress = opts.dirForceCompression
+	ctx.DirForceDecompress = opts.dirForceDecompression
```

Comment:
Something somewhere should reject `--compress`+`--decompress` at the same time. Probably already in `c/image/directory`, not necessarily here.